Assets/Mirage/Runtime/Messages.cs

```diff
@@ -18,10 +18,10 @@ public struct AddCharacterMessage { }
     [NetworkMessage]
     public struct SceneMessage
     {
-        public string scenePath;
+        public string MainActivateScene;
         // Normal = 0, LoadAdditive = 1, UnloadAdditive = 2
-        public SceneOperation sceneOperation;
-        public string[] additiveScenes;
+        public SceneOperation SceneOperation;
+        public string[] AdditiveScenes;
     }
 
     [NetworkMessage]
```

Comment:
we should rename the other message `ReadyMessage`, `NotReadyMessage` and `SceneReadyMessage` so they make more sense with the new `Player.SceneIsReady`

Reply:
I tried to do that but major stuff broken. I got confused what was what when I did that. Because ReadyMessage was used certain ReadyMessage are not same as ReadyMessage. I would rather do the changing of the other messages in a new pr separate of this one.